### DISCUSSION TITLE
Are the algorithms of the GitHub Trending public?

### DISCUSSION THREAD

**Discussion by xgdyp** [community/community#19470] [Other Feature Feedback, Questions, & Ideas] [answered]
I'm interested in the algorithms of GitHub Trending. But I haven't found any description of it. Is it public？

**Comment by itsgareth** [comment, marked as answer]
i have looked for something similar in the past to no avail, i'm pretty sure github keeps all of this information hidden.

**Comment by melvincwng** [comment]
@xgdyp, unfortunately I don't think the GitHub algorithms for trending page are open source.

Hope this clarifies.

**Comment by xgdyp** [comment]
Thanks！ @gwdawson @melvincwng

**Reply by melvincwng** [reply]
@xgdyp , pleasure to help!
